### ISSUE TITLE
surge - 2021-01-01 - 2021-04-30

### ISSUE THREAD

**Issue by bobobo199733** [bobobo199733/issues-metrics-tool-repo#14]
# Issue Metrics

| Metric | Average | Median | 90th percentile |
| --- | --- | --- | ---: |
| Time to first response | 9 days, 4:00:01 | 1:17:49 | 10 days, 8:12:24 |
| Time to close | 47 days, 5:47:21 | 4 days, 17:15:50 | 159 days, 4:05:54 |
| Time to answer | None | None | None |

| Metric | Count |
| --- | ---: |
| Number of items that remain open | 19 |
| Number of items closed | 312 |
| Total number of items created | 331 |

| Title | URL | Author | Time to first response | Time to close | Time to answer |
| --- | --- | --- | --- | --- | --- |
| XT Doesn't even start in debug mode on Windows | https://github.com/surge-synthesizer/surge/issues/4456 | mkruselj | 0:03:55 | 3 days, 0:03:49 | None |
| Tape FX in Surge takes ~2x more CPU on my machine compared to ChowTape | https://github.com/surge-synthesizer/surge/issues/4455 | VincyZed | 0:46:02 | 97 days, 22:59:37 | None |
| XT Patch database poof crash | https://github.com/surge-synthesizer/surge/issues/4453 | mkruselj | 0:12:54 | 3 days, 5:52:19 | None |
| SurgeXT/Linux Crash on click on LFO-Preset-Menu | https://github.com/surge-synthesizer/surge/issues/4445 | ghost | 0:24:27 | 7 days, 19:50:21 | None |
| Surge XT: Both filter Cutoff controls take 3 clicks to reset, instead of a standard double-click | https://github.com/surge-synthesizer/surge/issues/4441 | VincyZed | 6:01:21 | 1 day, 17:49:49 | None |
| Surge XT doesn't play Scene B when in Dual play mode | https://github.com/surge-synthesizer/surge/issues/4440 | VincyZed | 0:12:58 | 7:06:39 | None |
| JUCE SVG and UI issues | https://github.com/surge-synthesizer/surge/issues/4438 | VincyZed | 0:18:56 | 207 days, 12:37:33 | None |
| Lots of clicks when using SurgeEffectsBank | https://github.com/surge-synthesizer/surge/issues/4437 | feelware | 0:38:18 | 1 day, 20:21:11 | None |
| SurgeEffectsBank Can't Load in FL with .vst3 folder extension | https://github.com/surge-synthesizer/surge/issues/4434 | ramij294 | 0:03:48 | 1 day, 5:50:44 | None |
| XT Mousewheel is all over the place | https://github.com/surge-synthesizer/surge/issues/4432 | mkruselj | 15:25:12 | 28 days, 3:26:44 | None |
| Window oscillator is glitchy in continuous mode over wavetable frames | https://github.com/surge-synthesizer/surge/issues/4429 | baconpaul | 10 days, 0:57:54 | 14 days, 18:19:20 | None |
| Hovers not working on switches (XT) | https://github.com/surge-synthesizer/surge/issues/4423 | mkruselj | 3 days, 11:39:27 | 28 days, 1:49:22 | None |
| Remove deferred asset loader | https://github.com/surge-synthesizer/surge/issues/4422 | baconpaul | None | 4:11:10 | None |
| Add Accessibility Handlers to the TinyLittleIconButton (was: Expanded multi button menus) | https://github.com/surge-synthesizer/surge/issues/4418 | baconpaul | 19 days, 18:03:32 | 199 days, 14:59:44 | None |
| Root Folder Cleanups | https://github.com/surge-synthesizer/surge/issues/4415 | mkruselj | None | 4:31:22 | None |
| Finally properly separate UI and audio threads | https://github.com/surge-synthesizer/surge/issues/4413 | baconpaul | None | None | None |
| Linux xt deb file | https://github.com/surge-synthesizer/surge/issues/4412 | baconpaul | None | 8:06:17 | None |
| Rasperrry PI native didn't survive my CMake Shuffles | https://github.com/surge-synthesizer/surge/issues/4410 | baconpaul | None | 0:17:49 | None |
| MSEG Editor Rendering Issues | https://github.com/surge-synthesizer/surge/issues/4409 | baconpaul | 19 days, 18:25:10 | 70 days, 7:54:23 | None |
| JUCE vs EFVG Z-order | https://github.com/surge-synthesizer/surge/issues/4408 | baconpaul | None | 29 days, 19:36:57 | None |
| Menu Folder Checkmarks | https://github.com/surge-synthesizer/surge/issues/4407 | baconpaul | 29 days, 0:24:25 | 29 days, 1:27:23 | None |
| JUCE Threading Crash - investigate before closing | https://github.com/surge-synthesizer/surge/issues/4406 | baconpaul | None | 121 days, 3:18:33 | None |
| Reverse Typeins to JUCE Parameters | https://github.com/surge-synthesizer/surge/issues/4405 | baconpaul | None | 157 days, 21:08:40 | None |
| JUCE Ranged Param with Parameter range, or 0,1 | https://github.com/surge-synthesizer/surge/issues/4404 | baconpaul | None | 8 days, 8:28:24 | None |
| XT Parameter Mapping for the Macros | https://github.com/surge-synthesizer/surge/issues/4403 | baconpaul | None | 69 days, 5:44:10 | None |
| XT Zooming Problems | https://github.com/surge-synthesizer/surge/issues/4402 | baconpaul | None | 66 days, 5:19:07 | None |
| XT CMake Options | https://github.com/surge-synthesizer/surge/issues/4401 | baconpaul | None | 12 days, 22:21:34 | None |
| XT Mac Installer stubbornly refuses to install the .app | https://github.com/surge-synthesizer/surge/issues/4400 | baconpaul | None | 69 days, 4:03:05 | None |
| Can Surge XT run without a factory directory | https://github.com/surge-synthesizer/surge/issues/4398 | baconpaul | None | 108 days, 8:02:34 | None |
| Royal surge resizing with mouse not working | https://github.com/surge-synthesizer/surge/issues/4396 | ghost | 0:08:05 | 0:08:05 | None |
| Save oscillator and filter presets | https://github.com/surge-synthesizer/surge/issues/4395 | ghost | 0:20:44 | None | None |
| Add option to rename and delete presets | https://github.com/surge-synthesizer/surge/issues/4391 | Altarberg | 0:01:02 | 193 days, 3:10:53 | None |
| Finish the Scale Editor GUI | https://github.com/surge-synthesizer/surge/issues/4390 | baconpaul | 16:55:49 | 183 days, 3:04:07 | None |
| help popup for parameters | https://github.com/surge-synthesizer/surge/issues/4389 | ghost | 0:02:52 | 0:02:52 | None |
| preset management | https://github.com/surge-synthesizer/surge/issues/4388 | ghost | 0:01:57 | 0:02:00 | None |
| 1.9.0: UnitTestsDSP fails in modified arch build | https://github.com/surge-synthesizer/surge/issues/4387 | dvzrv | 0:41:18 | 21:27:41 | None |
| Wheel in Debug in JUCE throws up an assertion | https://github.com/surge-synthesizer/surge/issues/4386 | baconpaul | None | 39 days, 22:29:19 | None |
| JUCE menus and labels deal incorrectly with Emoji | https://github.com/surge-synthesizer/surge/issues/4385 | baconpaul | 86 days, 11:20:58 | 346 days, 23:50:14 | None |
| JUCE Keybindings in SurgeGUIEditor | https://github.com/surge-synthesizer/surge/issues/4383 | baconpaul | None | 120 days, 23:55:36 | None |
| XT About Box Copy and Paste of version info doesn't work | https://github.com/surge-synthesizer/surge/issues/4382 | baconpaul | None | 31 days, 2:02:07 | None |
| Set Custom User Data Folder in XT | https://github.com/surge-synthesizer/surge/issues/4378 | baconpaul | None | 120 days, 16:35:58 | None |
| ED's XT pedantikons | https://github.com/surge-synthesizer/surge/issues/4377 | mkruselj | 0:08:43 | 86 days, 17:36:54 | None |
| Toggling routing mode with mouse3, 4 or 5 when hovering over Alias additive editor doesn't work | https://github.com/surge-synthesizer/surge/issues/4374 | VincyZed | 0:08:54 | 2 days, 19:52:36 | None |
| Gradient leak in XT | https://github.com/surge-synthesizer/surge/issues/4372 | baconpaul | None | 12 days, 11:14:24 | None |
| Expose Trigger as a modulatable Twist parameter | https://github.com/surge-synthesizer/surge/issues/4370 | williamfields | 0:20:58 | 18 days, 5:38:33 | None |
| cannot load patch from file | https://github.com/surge-synthesizer/surge/issues/4368 | ghost | 0:03:12 | 2:42:04 | None |
| Bitwig Typing Keyboard hung notes | https://github.com/surge-synthesizer/surge/issues/4367 | ghost | 0:10:27 | 3:03:38 | None |
| Semimodular Envelope Trigger/Gate | https://github.com/surge-synthesizer/surge/issues/4366 | selenologist | 3:48:30 | 70 days, 12:09:39 | None |
| Sliders are missing on the 'Low Cut' and 'High Cut' widgets | https://github.com/surge-synthesizer/surge/issues/4360 | yurivict | 0:00:50 | 0:00:50 | None |
| No install target in cmake scripts | https://github.com/surge-synthesizer/surge/issues/4359 | yurivict | 0:03:21 | 14 days, 6:46:22 | None |
| Remove midifile dependency | https://github.com/surge-synthesizer/surge/issues/4358 | yurivict | 5:00:11 | 6:03:48 | None |
| Factory FX presets in a folder structure, in the binary | https://github.com/surge-synthesizer/surge/issues/4356 | mkruselj | 125 days, 3:24:16 | 125 days, 4:15:48 | None |
| Nodal routing for filter configuration | https://github.com/surge-synthesizer/surge/issues/4355 | mkruselj | 133 days, 3:50:53 | None | None |
| Extend Range on Filter Feedback | https://github.com/surge-synthesizer/surge/issues/4354 | selenologist | 5:25:20 | 23 days, 13:02:20 | None |
| JUCE UI memory leaks through Escape From VSTGUI | https://github.com/surge-synthesizer/surge/issues/4351 | baconpaul | None | 1 day, 18:07:37 | None |
| JUCE Cursor Hiding | https://github.com/surge-synthesizer/surge/issues/4350 | baconpaul | 49 days, 15:15:36 | 60 days, 16:54:31 | None |
| Rewrite the VU Meter as 100% JUCE code | https://github.com/surge-synthesizer/surge/issues/4348 | baconpaul | None | 32 days, 4:52:49 | None |
| VU Meters in Ardour in 1.9 are surprisingly expensive | https://github.com/surge-synthesizer/surge/issues/4347 | nicklaf | 0:06:22 | 4 days, 23:06:09 | None |
| Allow installation to D: or alt drive | https://github.com/surge-synthesizer/surge/issues/4345 | K8inLA | 4:03:21 | 4:03:21 | None |
| VST3 Parameter Menus in JUCE | https://github.com/surge-synthesizer/surge/issues/4342 | baconpaul | None | 69 days, 22:30:17 | None |
| Replace COptionMenu with juce::PopupMenu | https://github.com/surge-synthesizer/surge/issues/4341 | baconpaul | None | 62 days, 22:58:34 | None |
| Review TARGET_HEADLESS TARGET_JUCE TARGET_JUCE_UI | https://github.com/surge-synthesizer/surge/issues/4340 | baconpaul | None | 15 days, 13:53:26 | None |
| Remove the TARGET_(LV2&#124;AU&#124;VST2&#124;3) | https://github.com/surge-synthesizer/surge/issues/4339 | baconpaul | None | 3 days, 1:01:28 | None |
| JUCE Fonts | https://github.com/surge-synthesizer/surge/issues/4338 | baconpaul | None | 23:20:43 | None |
| Code Units we can Remove/Rewrite now we have JUCE | https://github.com/surge-synthesizer/surge/issues/4337 | baconpaul | None | 8 days, 23:05:45 | None |
| Surge XT Installers | https://github.com/surge-synthesizer/surge/issues/4336 | baconpaul | 16 days, 17:02:38 | 16 days, 16:32:02 | None |
| Surge XT Locations | https://github.com/surge-synthesizer/surge/issues/4335 | baconpaul | 7:55:44 | 5 days, 1:14:47 | None |
| A4 remapping fails on a specific Hz value | https://github.com/surge-synthesizer/surge/issues/4332 | kheptagon | 0:02:59 | 1:15:09 | None |
| Twist Oscillator chords type | https://github.com/surge-synthesizer/surge/issues/4326 | CurlyMC | 0:13:12 | 17 days, 22:44:25 | None |
| Implement Continuous Morph toggle for Wavetable oscillator | https://github.com/surge-synthesizer/surge/issues/4318 | mkruselj | None | 11 days, 9:36:29 | None |
| Crash when loading Exciter in Send FX slot 2 | https://github.com/surge-synthesizer/surge/issues/4313 | mkruselj | None | 1:33:04 | None |
| note advance mode for the step sequencer | https://github.com/surge-synthesizer/surge/issues/4308 | simulatedsystems | 9 days, 18:48:42 | 9 days, 18:48:42 | None |
| SurgeXT branches, versions, names, about screen, etc.... | https://github.com/surge-synthesizer/surge/issues/4304 | baconpaul | None | 149 days, 17:34:02 | None |
| Release Checklist for Surge XT 1.0 | https://github.com/surge-synthesizer/surge/issues/4303 | baconpaul | 247 days, 18:20:01 | 280 days, 17:23:49 | None |
| The "Don't Go Back" plan | https://github.com/surge-synthesizer/surge/issues/4300 | baconpaul | None | 8 days, 6:03:48 | None |
| Combulator behaves abnormally after being bypassed and re-enabled in this patch | https://github.com/surge-synthesizer/surge/issues/4295 | VincyZed | 0:21:18 | 1:12:34 | None |
| FX Diagram: clicking on units doesn't register the first time after clicking on something in a context menu | https://github.com/surge-synthesizer/surge/issues/4294 | VincyZed | 0:09:29 | 0:09:29 | None |
| note_to_pitch_ignoring_tuning | https://github.com/surge-synthesizer/surge/issues/4291 | baconpaul | None | 14 days, 7:46:20 | None |
| Pitch bend range and semitones in hertz | https://github.com/surge-synthesizer/surge/issues/4289 | muziker | 0:31:52 | 1 day, 20:40:54 | None |
| translating the manual | https://github.com/surge-synthesizer/surge/issues/4287 | ghost | 0:05:51 | 4 days, 17:53:13 | None |
| Implement a voice pitch modulator as alternates to high low key scene modulators and voice keytrack | https://github.com/surge-synthesizer/surge/issues/4286 | mkruselj | 128 days, 4:56:18 | None | None |
| FM3 Ratio Display isn't Bipolar on Extend | https://github.com/surge-synthesizer/surge/issues/4282 | baconpaul | None | 1:06:50 | None |
| Dynamic scale tunings | https://github.com/surge-synthesizer/surge/issues/4281 | muziker | 0:03:12 | 1:18:33 | None |
| Press [Esc] while using parameter slider: cancels and restores pre-drag value. | https://github.com/surge-synthesizer/surge/issues/4280 | j5v | 0:03:42 | 0:25:57 | None |
| VST2 plugins build error in CMakeLists.txt | https://github.com/surge-synthesizer/surge/issues/4278 | tank-trax | 0:09:18 | 0:08:57 | None |
| UI freeze after: paste to Scene B then LFO selection | https://github.com/surge-synthesizer/surge/issues/4274 | j5v | 0:58:19 | 2:04:16 | None |
| Custom installation folder !!!!!!! | https://github.com/surge-synthesizer/surge/issues/4266 | 2blackbar | 0:42:23 | 0:42:23 | None |
| Prior patches malfunctioning (possible Sine oscillator type initiation issue or out of bounds parameters) | https://github.com/surge-synthesizer/surge/issues/4260 | inigokennedy | 1:09:46 | 2:31:24 | None |
| Add distinction between all-notes-off and all-sound-off | https://github.com/surge-synthesizer/surge/issues/4256 | mkruselj | 1:18:23 | 92 days, 1:38:57 | None |
| Cannot compile on ARM platfomr(RPI4) | https://github.com/surge-synthesizer/surge/issues/4255 | phil123456 | 0:09:08 | 19 days, 16:28:47 | None |
| More GCC11 issues? -Werror=free-nonheap-object causes surge-fx build to fail | https://github.com/surge-synthesizer/surge/issues/4251 | plaimbock | 0:30:55 | 13:41:43 | None |
| Placeholder: Filter Subtype and so on with appears deactivated | https://github.com/surge-synthesizer/surge/issues/4250 | baconpaul | None | 21 days, 9:27:27 | None |
| Filter Type Automation doesn’t update subtype | https://github.com/surge-synthesizer/surge/issues/4248 | baconpaul | None | 1 day, 9:06:20 | None |
| The Sonic Transformation Soundset by HO for the Vember Audio - Surge | https://github.com/surge-synthesizer/surge/issues/4246 | Sonic13564 | 0:22:43 | 0:28:02 | None |
| copying alias with additive to another osc does not remember additive settings | https://github.com/surge-synthesizer/surge/issues/4244 | baconpaul | 6:24:18 | 12:54:35 | None |
| Eurorack Resample Strategy | https://github.com/surge-synthesizer/surge/issues/4235 | baconpaul | None | 5 days, 7:21:09 | None |
| Profile this patch before 1.9 | https://github.com/surge-synthesizer/surge/issues/4232 | baconpaul | 1 day, 12:22:31 | 5 days, 19:14:28 | None |
| That windows menu split thing | https://github.com/surge-synthesizer/surge/issues/4226 | baconpaul | 7:15:06 | 1 day, 4:40:45 | None |
| Init Templates for new Oscillators; FX Presets | https://github.com/surge-synthesizer/surge/issues/4223 | baconpaul | None | 19 days, 14:20:20 | None |
| Clear invalid modulations on unstream | https://github.com/surge-synthesizer/surge/issues/4211 | mkruselj | 2 days, 6:34:15 | 7 days, 23:12:39 | None |
| Modulator Envelope Presets Saving to LFO Folder | https://github.com/surge-synthesizer/surge/issues/4210 | bboxdave | None | 18 days, 6:53:57 | None |
| UI Tweaks for Alias Spectrum (was Separate storage for alias spectrum mode) | https://github.com/surge-synthesizer/surge/issues/4209 | baconpaul | 3 days, 11:49:46 | 4 days, 20:57:35 | None |
| Sudden pops and long sound dropouts occur in 1.9  beta | https://github.com/surge-synthesizer/surge/issues/4207 | Oortseam | 0:17:53 | 18:50:56 | None |
| Cannot compile on raspberry pi | https://github.com/surge-synthesizer/surge/issues/4206 | phil123456 | 0:03:37 | 5 days, 0:26:35 | None |
| Build failure: error: 'numeric_limits' is not a member of 'std' | https://github.com/surge-synthesizer/surge/issues/4205 | plaimbock | 0:28:29 | 3 days, 6:53:19 | None |
| Give MSEGs the ability to retrigger envelopes / additional envelope retrigger modes | https://github.com/surge-synthesizer/surge/issues/4201 | selenologist | 0:12:24 | 44 days, 15:45:30 | None |
| Filesystem exception if Documents/Surge isn’t readable | https://github.com/surge-synthesizer/surge/issues/4200 | baconpaul | None | 4:33:04 | None |
| Flutes 1 and 2 not working in Windows 10 | https://github.com/surge-synthesizer/surge/issues/4198 | ztnl | None | 0:01:59 | None |
| new 'rand' api doesn't work on some windows boxes (but not all) | https://github.com/surge-synthesizer/surge/issues/4197 | baconpaul | None | 6 days, 8:08:05 | None |
| When loading presets/patches/WTs from Explorer/Finder/DnD, check if path is indexed | https://github.com/surge-synthesizer/surge/issues/4196 | mkruselj | None | 10:28:22 | None |
| Store MIDI mapping to SurgeUserDefaults.xml | https://github.com/surge-synthesizer/surge/issues/4194 | mkruselj | 4 days, 21:42:15 | 6 days, 0:05:22 | None |
| no escape when zoom level larger than screen size | https://github.com/surge-synthesizer/surge/issues/4188 | ghost | 0:01:57 | 2:24:02 | None |
| Investigate making Window osc Morph continuous | https://github.com/surge-synthesizer/surge/issues/4184 | mkruselj | 1:27:56 | 32 days, 1:32:00 | None |
| Surge osc to filter routing and other stuff | https://github.com/surge-synthesizer/surge/issues/4183 | bigtonesounddesign | 2:30:45 | 403 days, 17:01:05 | None |
| Could these be storage->rand() instead? | https://github.com/surge-synthesizer/surge/issues/4180 | baconpaul | 12:54:55 | 15:14:50 | None |
| correlated_noise_o2mk2_suppliedrng in SurgeVoice | https://github.com/surge-synthesizer/surge/issues/4177 | baconpaul | 11 days, 21:07:58 | 5:09:40 | None |
| Loading FX preset with deactivated param doesn't work when changing FX types | https://github.com/surge-synthesizer/surge/issues/4176 | mkruselj | 5 days, 21:58:03 | 6 days, 3:22:54 | None |
| User Favourites | https://github.com/surge-synthesizer/surge/issues/4173 | Eyal2 | 0:38:25 | 1:21:50 | None |
| Questions regarding previous issue | https://github.com/surge-synthesizer/surge/issues/4172 | mexanik619 | 1:04:58 | 5 days, 0:34:07 | None |
| Tuning Mode Switch discards Tuning | https://github.com/surge-synthesizer/surge/issues/4170 | baconpaul | None | 23:41:48 | None |
| Pull some new AirWindows into 1.9 | https://github.com/surge-synthesizer/surge/issues/4160 | baconpaul | None | 1:06:53 | None |
| Fix limit_range definition; remove basic_dsp/Max and Min; etc.... | https://github.com/surge-synthesizer/surge/issues/4153 | baconpaul | None | 45 days, 9:37:00 | None |
| Let different channels transpose octaves for microtuning support | https://github.com/surge-synthesizer/surge/issues/4151 | battaglia01 | 4:03:05 | 139 days, 10:05:57 | None |
| LFO Rate parameter doesn't go transparent anymore when deactivated | https://github.com/surge-synthesizer/surge/issues/4145 | VincyZed | 0:07:43 | 0:31:12 | None |
| libSampleRate cmake file is spamming up the target list with extra goop | https://github.com/surge-synthesizer/surge/issues/4136 | baconpaul | None | 4:01:15 | None |
| JUCE has late samplerate startup - address | https://github.com/surge-synthesizer/surge/issues/4132 | baconpaul | None | 47 days, 21:06:15 | None |
| KBM .name doesn't store in daw extra or patch | https://github.com/surge-synthesizer/surge/issues/4122 | baconpaul | None | 3:39:21 | None |
| Scene Hard Clip is off by +4dB | https://github.com/surge-synthesizer/surge/issues/4117 | ghost | 0:33:44 | 4 days, 0:28:27 | None |
| Replace Eurorack submodule with local copy of just what we use | https://github.com/surge-synthesizer/surge/issues/4115 | mkruselj | 2 days, 15:19:29 | 2 days, 17:43:43 | None |
| Constant crashes in Bitwig (Kubuntu) | https://github.com/surge-synthesizer/surge/issues/4113 | alexamy | 0:08:02 | 0:08:29 | None |
| Dynamic Deactivated Query | https://github.com/surge-synthesizer/surge/issues/4109 | baconpaul | None | 5:38:51 | None |
| Re-work kBiPolar | https://github.com/surge-synthesizer/surge/issues/4108 | baconpaul | None | 0:39:28 | None |
| LFO waveform ghost aliasing | https://github.com/surge-synthesizer/surge/issues/4100 | mkruselj | 0:56:23 | 5:39:36 | None |
| EffectGrid Click stolen on Windows only after loading an FX preset | https://github.com/surge-synthesizer/surge/issues/4098 | mkruselj | 6:58:22 | 16 days, 17:33:10 | None |
| SurgeFX Init can ask for incorrect Note | https://github.com/surge-synthesizer/surge/issues/4097 | ghost | 1:56:59 | 51 days, 1:59:41 | None |
| Idiocy (no offense) | https://github.com/surge-synthesizer/surge/issues/4096 | Jazzmaster101 | 0:01:33 | 0:01:53 | None |
| Stuck notes in Ableton 11 | https://github.com/surge-synthesizer/surge/issues/4095 | jomicof | 0:16:34 | 0:16:34 | None |
| Click on Surge parameter and it crashes Ableton live 10 | https://github.com/surge-synthesizer/surge/issues/4094 | fiddlestickzmuzik | 4:52:32 | 2 days, 9:16:10 | None |
| SurgeEffectsBank quantizes all parameters to the nearest percent | https://github.com/surge-synthesizer/surge/issues/4088 | bit-101010 | 0:34:38 | 7:51:08 | None |
| UserPref Keys should be variables | https://github.com/surge-synthesizer/surge/issues/4085 | baconpaul | 41 days, 22:47:46 | 41 days, 22:47:46 | None |
| Story PolyCount in the Patch | https://github.com/surge-synthesizer/surge/issues/4084 | baconpaul | None | 1:23:38 | None |
| Carla Volume knob does not affect volume of Surge | https://github.com/surge-synthesizer/surge/issues/4082 | muziker | 5:02:25 | 22:21:49 | None |
| continous open/close of Surge GUI is able to max out ram | https://github.com/surge-synthesizer/surge/issues/4081 | ghost | 0:02:34 | 23:16:13 | None |
| Can't change installation directory | https://github.com/surge-synthesizer/surge/issues/4079 | molecooler | 0:14:10 | 0:14:10 | None |
| Tempo as modulation source | https://github.com/surge-synthesizer/surge/issues/4075 | baconpaul | 389 days, 11:15:37 | 389 days, 6:06:47 | None |
| Tempo-Locked Base Tuning | https://github.com/surge-synthesizer/surge/issues/4074 | baconpaul | None | None | None |
| How to install surge-fx ? Not in README | https://github.com/surge-synthesizer/surge/issues/4067 | muziker | 0:15:52 | 5 days, 2:43:31 | None |
| Give us the option to choose the installation directory!! | https://github.com/surge-synthesizer/surge/issues/4066 | SirTwsted | 0:01:38 | 0:04:45 | None |
| FM feedback default to extended range | https://github.com/surge-synthesizer/surge/issues/4064 | williamfields | 0:18:13 | 3:17:24 | None |
| FX drop downlist does not scroll | https://github.com/surge-synthesizer/surge/issues/4062 | muziker | 2:17:55 | 7:12:08 | None |
| LV2 plugin in Carla removes effects from bay when toggling the GUI editor | https://github.com/surge-synthesizer/surge/issues/4061 | muziker | 2:37:04 | 41 days, 8:56:08 | None |
| CMenuAsSlider -> kNoPopup which Sticks | https://github.com/surge-synthesizer/surge/issues/4059 | baconpaul | None | 0:33:48 | None |
| Hardclip cleanups | https://github.com/surge-synthesizer/surge/issues/4052 | K0rrid0r | None | 9:15:15 | None |
| GUI is not updating when I turn the knobs on my midi controller | https://github.com/surge-synthesizer/surge/issues/4050 | kanduvisla | 0:53:35 | 167 days, 4:11:23 | None |
| recall issue with Samplitude | https://github.com/surge-synthesizer/surge/issues/4047 | otison73 | 1:17:15 | 196 days, 0:03:17 | None |
| XT Native Tuning Improvements | https://github.com/surge-synthesizer/surge/issues/4046 | baconpaul | None | 116 days, 4:33:52 | None |
| Phaser crash with noise source | https://github.com/surge-synthesizer/surge/issues/4043 | baconpaul | None | 1:52:10 | None |
| gui not working for all version of surge including nightly version | https://github.com/surge-synthesizer/surge/issues/4042 | mexanik619 | 1:29:08 | 5 days, 0:28:11 | None |
| Beat frequency detuning | https://github.com/surge-synthesizer/surge/issues/4041 | triss | 0:18:00 | None | None |
| Sustain pedal not working in Ableton Live 11 | https://github.com/surge-synthesizer/surge/issues/4039 | erichaleymusic | 0:11:18 | 0:34:09 | None |
| UI vs Audio Thread Access under Extreme Automation | https://github.com/surge-synthesizer/surge/issues/4038 | K0rrid0r | 14 days, 23:57:00 | 191 days, 5:50:55 | None |
| Treemonster wrap up | https://github.com/surge-synthesizer/surge/issues/4034 | baconpaul | None | 18 days, 23:49:50 | None |
| Oscillator Common Code refactoring | https://github.com/surge-synthesizer/surge/issues/4032 | baconpaul | None | 7:52:52 | None |
| Hardclip Global Volume at -1, 1 | https://github.com/surge-synthesizer/surge/issues/4031 | K0rrid0r | None | 4 days, 11:46:15 | None |
| JUCE LV2 won't build in unattended/non-XWindows pipeline builds | https://github.com/surge-synthesizer/surge/issues/4030 | baconpaul | 46 days, 9:58:23 | 49 days, 2:27:41 | None |
| Preset switching bug | https://github.com/surge-synthesizer/surge/issues/4029 | Engylizium | 1:32:19 | 20 days, 4:18:27 | None |
| AU and VST2 advertise MPE support | https://github.com/surge-synthesizer/surge/issues/4027 | baconpaul | None | 6 days, 9:49:49 | None |
| My patch suddenly went completely silent | https://github.com/surge-synthesizer/surge/issues/4024 | unfa | 0:46:22 | 1:48:28 | None |
| Osc Drift - separate for each oscillator | https://github.com/surge-synthesizer/surge/issues/4022 | mkruselj | None | None | None |
| All Tests have Bounded Output Test | https://github.com/surge-synthesizer/surge/issues/4019 | baconpaul | None | None | None |
| Delay cut-off | https://github.com/surge-synthesizer/surge/issues/4016 | jocphx | 0:19:43 | 302 days, 3:08:45 | None |
| Spend a day re-running leak check tools | https://github.com/surge-synthesizer/surge/issues/4014 | baconpaul | 3 days, 0:02:15 | 36 days, 6:07:12 | None |
| Stuck MIDI Notes | https://github.com/surge-synthesizer/surge/issues/4013 | nedrob | 0:05:14 | 13 days, 19:46:20 | None |
| Stuck midi Notes | https://github.com/surge-synthesizer/surge/issues/4012 | nedrob | None | 0:11:40 | None |
| Keytracking toggle for all oscillators | https://github.com/surge-synthesizer/surge/issues/4005 | noyzelab | 0:17:44 | 11:13:19 | None |
| 1.8.1: Unable to build tarball if git not present | https://github.com/surge-synthesizer/surge/issues/4000 | dvzrv | 0:11:03 | 1:34:02 | None |
| tun button should light up if odd mts is in control | https://github.com/surge-synthesizer/surge/issues/3994 | baconpaul | None | 0:40:29 | None |
| Better KBM skip note in oddmts mode | https://github.com/surge-synthesizer/surge/issues/3988 | baconpaul | None | 1:59:27 | None |
| Surge XT crash on Linux | https://github.com/surge-synthesizer/surge/issues/3987 | baconpaul | 15:42:34 | 16:03:10 | None |
| Menu tweak 12 note to 12 tone equal | https://github.com/surge-synthesizer/surge/issues/3986 | baconpaul | 1:31:04 | 2:08:02 | None |
| Note to pitch should be loglinear interpolation | https://github.com/surge-synthesizer/surge/issues/3985 | baconpaul | None | 4:45:38 | None |
| KBM toggle broke when tatm was introduced | https://github.com/surge-synthesizer/surge/issues/3984 | baconpaul | None | 19 days, 7:07:55 | None |
| Nightly AU is kinda hosed in Reaper on Mac | https://github.com/surge-synthesizer/surge/issues/3981 | baconpaul | None | 0:19:06 | None |
| Arty reports nightly crashes bitwig windows 10 | https://github.com/surge-synthesizer/surge/issues/3979 | baconpaul | 1:47:25 | 3:48:01 | None |
| Debian package should specify minimum library versions | https://github.com/surge-synthesizer/surge/issues/3956 | x42 | 0:48:12 | 4 days, 2:26:40 | None |
| Implement "Suspend" properly on all FX | https://github.com/surge-synthesizer/surge/issues/3950 | mkruselj | 10:39:18 | 6 days, 7:42:18 | None |
| Integrate Dragonfly plate reverb and early reflections | https://github.com/surge-synthesizer/surge/issues/3948 | mkruselj | 5 days, 3:55:39 | 227 days, 23:37:37 | None |
| Endpoint MSEG fix draws fake line | https://github.com/surge-synthesizer/surge/issues/3940 | baconpaul | None | 5:15:56 | None |
| DPW Deform Reset | https://github.com/surge-synthesizer/surge/issues/3929 | baconpaul | None | 0:50:36 | None |
| PR Pipeline Speedup | https://github.com/surge-synthesizer/surge/issues/3912 | baconpaul | None | 11:35:19 | None |
| DPW Tracking Issue | https://github.com/surge-synthesizer/surge/issues/3910 | baconpaul | None | 1 day, 13:39:15 | None |
| Lessen CPU load on long note release times | https://github.com/surge-synthesizer/surge/issues/3909 | muziker | 0:32:38 | 61 days, 8:52:05 | None |
| Voice unison | https://github.com/surge-synthesizer/surge/issues/3895 | baconpaul | 0:06:27 | None | None |
| Add MSEG segment endpoint to the curve path | https://github.com/surge-synthesizer/surge/issues/3893 | mkruselj | 12:32:06 | 13:45:31 | None |
| Weird pitch bend > deactivated LFO phase modulation | https://github.com/surge-synthesizer/surge/issues/3892 | mkruselj | None | 0:03:54 | None |
| Update all the Jacky patches to new TATM mode | https://github.com/surge-synthesizer/surge/issues/3891 | baconpaul | None | 3 days, 4:15:52 | None |
| Skin component model additions | https://github.com/surge-synthesizer/surge/issues/3889 | baconpaul | 1 day, 13:14:21 | 523 days, 10:34:06 | None |
| Tuning to MIDI_BLAH for 19 default | https://github.com/surge-synthesizer/surge/issues/3883 | baconpaul | None | 1:02:20 | None |
| Ringout when changing FX happens pre initialization | https://github.com/surge-synthesizer/surge/issues/3878 | baconpaul | None | 4 days, 4:50:04 | None |
| LFO max amplitude ghost looks different compared to actual waveform | https://github.com/surge-synthesizer/surge/issues/3877 | mkruselj | 2:48:12 | 4:23:19 | None |
| The CEffectGrid should be drawn in code, not skins, with skin colors | https://github.com/surge-synthesizer/surge/issues/3873 | baconpaul | 0:30:09 | 11 days, 23:49:41 | None |
| Skin: Move to version 2; add version aware APIs | https://github.com/surge-synthesizer/surge/issues/3860 | baconpaul | None | 13:47:52 | None |
| Using <groups> causes some layout issues | https://github.com/surge-synthesizer/surge/issues/3855 | itsmedavep | 7 days, 3:27:36 | 7 days, 4:30:41 | None |
| KBM in Modulation-Out-Of-Tuning mode | https://github.com/surge-synthesizer/surge/issues/3853 | baconpaul | None | 1 day, 18:58:57 | None |
| A few quality of life requests for skin devs | https://github.com/surge-synthesizer/surge/issues/3846 | itsmedavep | None | 8 days, 0:51:07 | None |
| Reaper: Setting the zoom to 175% on Royal skin causes black borders | https://github.com/surge-synthesizer/surge/issues/3844 | Altarberg | 0:18:10 | 4:42:45 | None |
| Content for the 1.9 manual update | https://github.com/surge-synthesizer/surge/issues/3842 | baconpaul | 11 days, 1:06:14 | 34 days, 5:27:57 | None |
| Show modulator in display when amplitude is 0 | https://github.com/surge-synthesizer/surge/issues/3838 | magnetophon | 0:05:18 | 4 days, 23:26:21 | None |
| Slider label hiding and skin label listener | https://github.com/surge-synthesizer/surge/issues/3835 | mkruselj | 1:05:54 | 3:42:54 | None |
| UI zoom level problems in REAPER | https://github.com/surge-synthesizer/surge/issues/3834 | williamfields | 0:18:10 | 8 days, 22:37:18 | None |
| Type frequency as note name | https://github.com/surge-synthesizer/surge/issues/3831 | magnetophon | 4 days, 16:10:02 | 4 days, 16:38:27 | None |
| Tuning blank line fails to parse | https://github.com/surge-synthesizer/surge/issues/3830 | baconpaul | None | 0:51:35 | None |
| Tuning aware filters | https://github.com/surge-synthesizer/surge/issues/3827 | baconpaul | None | 6 days, 20:44:51 | None |
| Tuning Modes | https://github.com/surge-synthesizer/surge/issues/3826 | baconpaul | None | 1 day, 14:53:53 | None |
| 182 combulator | https://github.com/surge-synthesizer/surge/issues/3819 | baconpaul | None | 1:09:41 | None |
| Bitwig resolution offset maybe? | https://github.com/surge-synthesizer/surge/issues/3818 | baconpaul | None | 41 days, 23:13:58 | None |
| No sound make latch patches in factory or 3rd part library please! | https://github.com/surge-synthesizer/surge/issues/3816 | itsmedavep | 0:43:02 | 15:26:38 | None |
| Assign image to a label | https://github.com/surge-synthesizer/surge/issues/3815 | mkruselj | None | 6 days, 8:04:10 | None |
| Parameterize the patch browser layout | https://github.com/surge-synthesizer/surge/issues/3814 | itsmedavep | 17:19:47 | 533 days, 15:41:13 | None |
| Not all of the <controls> take img= references | https://github.com/surge-synthesizer/surge/issues/3813 | itsmedavep | None | 10 days, 22:26:06 | None |
| Combulator expansion | https://github.com/surge-synthesizer/surge/issues/3811 | mkruselj | 1:19:38 | None | None |
| Add Mutable Instruments Plaits as oscillator type | https://github.com/surge-synthesizer/surge/issues/3810 | mkruselj | 0:05:20 | 44 days, 2:45:01 | None |
| Go sweep for constness and stuff | https://github.com/surge-synthesizer/surge/issues/3808 | baconpaul | None | 200 days, 9:55:58 | None |
| Window Oscillator Unison 1 broken | https://github.com/surge-synthesizer/surge/issues/3798 | baconpaul | None | 0:37:47 | None |
| Increase the hitzone for the macro slider | https://github.com/surge-synthesizer/surge/issues/3791 | mkruselj | None | 7:51:56 | None |
| Ardour can’t spawn zenity | https://github.com/surge-synthesizer/surge/issues/3790 | unfa | 0:03:56 | 4:06:32 | None |
| Sine Oscillator Aliasing with LFO on Pitch | https://github.com/surge-synthesizer/surge/issues/3784 | unfa | 0:17:42 | None | None |
| Bitwig and Modulation | https://github.com/surge-synthesizer/surge/issues/3783 | baconpaul | 2 days, 10:54:29 | 2 days, 12:23:03 | None |
| Filter glyph placement | https://github.com/surge-synthesizer/surge/issues/3782 | baconpaul | 5:50:02 | 14 days, 16:40:10 | None |
| 19 SFX Bank gonna need some work | https://github.com/surge-synthesizer/surge/issues/3779 | baconpaul | None | 74 days, 12:18:19 | None |
| Crash on LPX with Combulator | https://github.com/surge-synthesizer/surge/issues/3778 | baconpaul | None | 4 days, 18:40:43 | None |
| Patches using Single Trigger do not work correctly with Ableton Live 11 Beta and MPE | https://github.com/surge-synthesizer/surge/issues/3776 | sourcebox | 0:02:41 | 37 days, 6:35:50 | None |
| Combulator - outstanding items | https://github.com/surge-synthesizer/surge/issues/3773 | baconpaul | None | 11 days, 1:02:14 | None |
| Processor info on about on windows | https://github.com/surge-synthesizer/surge/issues/3771 | baconpaul | None | 5:27:30 | None |
| Flanger neuron code sharing | https://github.com/surge-synthesizer/surge/issues/3770 | baconpaul | 1 day, 4:49:15 | 416 days, 8:20:28 | None |
| Cross platform LV2 builds | https://github.com/surge-synthesizer/surge/issues/3768 | dromer | 0:11:08 | 0:11:08 | None |
| SS and Mod reported on KVR | https://github.com/surge-synthesizer/surge/issues/3760 | baconpaul | 3:18:39 | 6 days, 13:23:09 | None |
| Curves for macros and aftetouch, pitchbend etc... | https://github.com/surge-synthesizer/surge/issues/3758 | arumihsnek | 0:12:14 | 0:13:35 | None |
| Skin installation | https://github.com/surge-synthesizer/surge/issues/3757 | mkruselj | None | 211 days, 20:32:46 | None |
| Nimbus - Mutable Instruments Clouds inspired granular effect | https://github.com/surge-synthesizer/surge/issues/3756 | baconpaul | 1 day, 0:57:27 | 52 days, 0:39:14 | None |
| LFO for Comb Frequency in Neuron | https://github.com/surge-synthesizer/surge/issues/3755 | baconpaul | 1:21:03 | 1 day, 18:41:29 | None |
| Make patch jog buttons continue to next or previous folder | https://github.com/surge-synthesizer/surge/issues/3749 | mkruselj | 2:20:52 | 11 days, 23:48:18 | None |
| Royal Surge Skin: higher zoom levels | https://github.com/surge-synthesizer/surge/issues/3746 | mzed | 0:51:13 | 0:51:33 | None |
| Release Checklist for version 1.9 | https://github.com/surge-synthesizer/surge/issues/3745 | baconpaul | None | 82 days, 10:40:33 | None |
| BBD ensemble/chorus | https://github.com/surge-synthesizer/surge/issues/3743 | mkruselj | 0:02:50 | 50 days, 16:42:41 | None |
| Surge VST3 Crash in Ardour 6.5 after fiddling with some controls (GDB backtrace!) | https://github.com/surge-synthesizer/surge/issues/3741 | unfa | 13 days, 4:15:32 | 29 days, 18:05:10 | None |
| AU in Logic has too granular parameter changes under automation | https://github.com/surge-synthesizer/surge/issues/3740 | baconpaul | None | 53 days, 21:51:55 | None |
| Port to JUCE book of work | https://github.com/surge-synthesizer/surge/issues/3737 | baconpaul | None | 87 days, 3:31:08 | None |
| MSEG less zoomed when Rate is deactivated. | https://github.com/surge-synthesizer/surge/issues/3731 | magnetophon | 17:08:00 | 18:04:28 | None |
| fractional ratios in FM oscillators | https://github.com/surge-synthesizer/surge/issues/3730 | magnetophon | 0:25:26 | 0:37:38 | None |
| Optimize 11-band graphic EQ with SSE | https://github.com/surge-synthesizer/surge/issues/3726 | mkruselj | None | 7 days, 21:27:21 | None |
| Long category and patch names overlap | https://github.com/surge-synthesizer/surge/issues/3725 | MD-78 | 0:06:00 | 18:46:55 | None |
| Bipolar feedback in Comb filters | https://github.com/surge-synthesizer/surge/issues/3724 | magnetophon | 0:04:37 | None | None |
| Reverb 1 decay parameter doesn't load from FX presets | https://github.com/surge-synthesizer/surge/issues/3717 | TheNerdyMusicGuy | 1:36:45 | 159 days, 7:32:16 | None |
| Optionally remember osc parameter values per osc type | https://github.com/surge-synthesizer/surge/issues/3716 | mkruselj | None | 212 days, 3:51:01 | None |
| Lift MSEG LFO edit mode limitation that start node value = end node value | https://github.com/surge-synthesizer/surge/issues/3714 | mkruselj | None | 26 days, 16:26:30 | None |
| Copying a wavetable osc looses it's place in the WT list | https://github.com/surge-synthesizer/surge/issues/3710 | magnetophon | 1:54:53 | 3 days, 0:28:28 | None |
| More state information in macro sliders. | https://github.com/surge-synthesizer/surge/issues/3709 | magnetophon | 0:08:57 | 14 days, 5:36:45 | None |
| Minor drawing bug in S-Curve | https://github.com/surge-synthesizer/surge/issues/3708 | magnetophon | 0:15:10 | 2:52:03 | None |
| Erratic behavior with Wacom tablet | https://github.com/surge-synthesizer/surge/issues/3701 | Brier1104 | 1:57:07 | 2:21:52 | None |
| MSEG hover path drawing bug when scrolling | https://github.com/surge-synthesizer/surge/issues/3697 | mkruselj | 3:48:11 | 13:39:04 | None |
| Modulation assignments to FX parameters not carried over when moving FX via DnD | https://github.com/surge-synthesizer/surge/issues/3694 | mkruselj | None | 5:03:16 | None |
| iOS version? | https://github.com/surge-synthesizer/surge/issues/3693 | tomduncalf | 0:23:44 | 0:23:44 | None |
| Windows protected your PC warning? | https://github.com/surge-synthesizer/surge/issues/3692 | hems | 2:25:34 | 2:26:05 | None |
| Surge not loading osx10.11 logic 9.1 | https://github.com/surge-synthesizer/surge/issues/3691 | 88pidhpidh | 4:53:55 | 242 days, 10:51:19 | None |
| Automatically choose best fit zoom level on very first load | https://github.com/surge-synthesizer/surge/issues/3686 | mkruselj | 3 days, 12:51:19 | 161 days, 7:02:32 | None |
| Airwindows Vari-Mu creates aliasing | https://github.com/surge-synthesizer/surge/issues/3684 | Vospi | 4:45:44 | 4:44:40 | None |
| Sound sometimes cuts out when switching to Wide mode | https://github.com/surge-synthesizer/surge/issues/3683 | TheNerdyMusicGuy | 6:37:11 | 816 days, 14:31:42 | None |
| Double activation from a single click of the left/right patch arrows | https://github.com/surge-synthesizer/surge/issues/3682 | j5v | 71 days, 7:15:40 | 71 days, 7:15:40 | None |
| Mod amount tooltip sometimes stays on screen instead of timing out | https://github.com/surge-synthesizer/surge/issues/3680 | VincyZed | 0:23:02 | 2:21:14 | None |
| Status Issue - please read before PR or Merge (updated Apr 21) | https://github.com/surge-synthesizer/surge/issues/3677 | baconpaul | None | 92 days, 5:59:05 | None |
| Arpeggiator | https://github.com/surge-synthesizer/surge/issues/3675 | MrHammyMark | 1:06:31 | None | None |
| Cannot create a category that contains (I guess) Unicode chars | https://github.com/surge-synthesizer/surge/issues/3673 | mkruselj | 6:51:21 | 22 days, 5:18:36 | None |
| Non-Unicode author names stored in FXPs | https://github.com/surge-synthesizer/surge/issues/3672 | mkruselj | 22 days, 3:38:14 | 22 days, 3:38:23 | None |
| Modulation on TempoSync LFO Rate | https://github.com/surge-synthesizer/surge/issues/3671 | baconpaul | 0:09:20 | 2:39:04 | None |
| Formant oscillators | https://github.com/surge-synthesizer/surge/issues/3670 | magnetophon | None | None | None |
| Can we author Surge oscillators or FX in Faust? | https://github.com/surge-synthesizer/surge/issues/3669 | baconpaul | 10:15:27 | None | None |
| 1.8.0: error: 'magnify' marked 'override' but does not override any member functions    virtual bool magnify(CPoint& where, float amount) override | https://github.com/surge-synthesizer/surge/issues/3668 | yurivict | 0:06:00 | 0:31:50 | None |
| Question: i have an AMD processor, the about windows displays "Intel CPU", is that normal behaviour? | https://github.com/surge-synthesizer/surge/issues/3666 | baconpaul | None | 1:41:19 | None |
| Something is wrong with mouse on Linux | https://github.com/surge-synthesizer/surge/issues/3662 | baconpaul | 6 days, 13:06:16 | 81 days, 7:30:47 | None |
| Harrison MixBus Mac problems | https://github.com/surge-synthesizer/surge/issues/3661 | baconpaul | None | 1 day, 0:08:23 | None |
| Update / expand 18 content with refresh of some patches and addition of stuff we didn't get to | https://github.com/surge-synthesizer/surge/issues/3660 | baconpaul | 8 days, 18:37:48 | 8 days, 18:37:49 | None |
| Oscillator waveform can go way past top/bottom of the display in some cases | https://github.com/surge-synthesizer/surge/issues/3658 | mkruselj | 1:44:34 | 96 days, 3:14:51 | None |
| DnD or Load WT from file always switches osc type to Wavetable | https://github.com/surge-synthesizer/surge/issues/3657 | mkruselj | 6:44:14 | 12:24:22 | None |
| Lato Not Installed warning only needs to appear once | https://github.com/surge-synthesizer/surge/issues/3654 | baconpaul | None | 1 day, 0:58:50 | None |
| Velocity and Gain Labels (or sliders) switched in 18 classic and dark skin | https://github.com/surge-synthesizer/surge/issues/3653 | baconpaul | 1:54:44 | 1:58:30 | None |
| Release Checklist for version 1.8.1 | https://github.com/surge-synthesizer/surge/issues/3652 | baconpaul | None | 9 days, 18:05:25 | None |
| Click on Loop in Reaper | https://github.com/surge-synthesizer/surge/issues/3651 | DimiGuitar | 0:36:20 | 23:40:20 | None |
| Cap missing from Width slider in Output section. | https://github.com/surge-synthesizer/surge/issues/3650 | ContemporaryInsanity | 0:12:23 | 0:23:06 | None |
| Surge sometimes crashes when changing presets | https://github.com/surge-synthesizer/surge/issues/3648 | TheNerdyMusicGuy | 2:56:26 | 4 days, 13:26:35 | None |
| FX Send as a per voice parameter | https://github.com/surge-synthesizer/surge/issues/3645 | PopeKehoeIII | 23:52:59 | None | None |
| Look for resources in ${CMAKE_INSTALL_PREFIX} | https://github.com/surge-synthesizer/surge/issues/3641 | magnetophon | 0:19:00 | 26 days, 1:57:56 | None |
| "Copy With Modulation" for everything | https://github.com/surge-synthesizer/surge/issues/3639 | magnetophon | 1 day, 6:38:17 | 1 day, 6:38:17 | None |
| Wavetable name does not get copied when copying oscillators | https://github.com/surge-synthesizer/surge/issues/3637 | magnetophon | None | 1:11:15 | None |
| Option to label the LFO display in musical time | https://github.com/surge-synthesizer/surge/issues/3635 | magnetophon | 0:16:22 | 0:31:42 | None |
| Last minute 1.8 things | https://github.com/surge-synthesizer/surge/issues/3634 | baconpaul | 13:36:01 | 13:46:48 | None |
| SMSEG Gate Mode Retrigger skips attacks | https://github.com/surge-synthesizer/surge/issues/3631 | baconpaul | None | 2:14:53 | None |
| Integrate LuaJIT with Surge | https://github.com/surge-synthesizer/surge/issues/3627 | baconpaul | 6 days, 11:22:53 | 6 days, 11:22:53 | None |
| Zoom window with error appear on loading the synth on MacBook Air M1 | https://github.com/surge-synthesizer/surge/issues/3623 | Stansom | 0:12:32 | 5:59:55 | None |
| Mousewheel on modbuttons - flip between modulator alternates (if available) | https://github.com/surge-synthesizer/surge/issues/3622 | mkruselj | 2 days, 9:49:52 | 225 days, 2:34:44 | None |
| Copy/Paste doesn’t take macros and in some cases should | https://github.com/surge-synthesizer/surge/issues/3621 | baconpaul | 0:17:25 | None | None |
| FM4 oscillator | https://github.com/surge-synthesizer/surge/issues/3619 | mkruselj | 0:41:19 | None | None |
| OSC and LFO display animation: questions, ideas, clarifications and solutions | https://github.com/surge-synthesizer/surge/issues/3618 | VincyZed | 3 days, 6:53:39 | None | None |
| 3D Waveform for OSC (when morphing waveforms) | https://github.com/surge-synthesizer/surge/issues/3617 | artyfunk | 17:48:32 | 427 days, 19:43:12 | None |
| Surge Crashes in linux wine environment when loading into FL Studio 2020 | https://github.com/surge-synthesizer/surge/issues/3616 | BruhMomentInifity | 0:19:39 | 15:13:54 | None |
| Link left/right times in delay FX | https://github.com/surge-synthesizer/surge/issues/3613 | artyfunk | 8:59:54 | 18 days, 0:35:14 | None |
| PNG Deferred Load | https://github.com/surge-synthesizer/surge/issues/3607 | baconpaul | None | 14:30:36 | None |
| Handle the CHSwitch2 Special Case in SkinSupport::reloadSkin | https://github.com/surge-synthesizer/surge/issues/3605 | baconpaul | None | 36 days, 19:22:38 | None |
| CHSwitch2 should support custom skin classes | https://github.com/surge-synthesizer/surge/issues/3602 | mkruselj | 9:35:07 | 9:41:48 | None |
| Mono note priority mode not preserved on copy/paste | https://github.com/surge-synthesizer/surge/issues/3599 | PopeKehoeIII | 11:42:01 | 12:05:24 | None |
| Global keytrack modulators not always updating properly | https://github.com/surge-synthesizer/surge/issues/3597 | PopeKehoeIII | 0:14:16 | 8:06:16 | None |
| M/S transform effect in FX or scene output section | https://github.com/surge-synthesizer/surge/issues/3596 | Oortseam | 3:12:11 | 265 days, 2:00:14 | None |
| Allow users to set the location of the Factory Data Folder | https://github.com/surge-synthesizer/surge/issues/3593 | maxxatgit | 2:52:44 | 21:02:12 | None |
| Surge, FileSystem, C++-17, etc | https://github.com/surge-synthesizer/surge/issues/3589 | baconpaul | 12 days, 22:03:24 | 30 days, 18:36:24 | None |
| Add ChowDSP Gated Recurrent Distortion | https://github.com/surge-synthesizer/surge/issues/3587 | jatinchowdhury18 | None | 21 days, 20:30:15 | None |
| Add ChowDSP Aural Exciter | https://github.com/surge-synthesizer/surge/issues/3586 | jatinchowdhury18 | 23 days, 15:38:13 | 29 days, 19:55:23 | None |
| Add ChowDSP ChowTape | https://github.com/surge-synthesizer/surge/issues/3585 | jatinchowdhury18 | 0:03:22 | 43 days, 4:38:28 | None |
| Add ChowDSP CHOW | https://github.com/surge-synthesizer/surge/issues/3584 | jatinchowdhury18 | 0:03:34 | 25 days, 1:58:21 | None |
| Wavetable needs a new Format for .wav files. | https://github.com/surge-synthesizer/surge/issues/3583 | artyfunk | 2:29:02 | 18:49:25 | None |
| Don't save MIDI learn to snapshot; don't MIDI learn both scenes; handle automation in Dual/Split etc properly | https://github.com/surge-synthesizer/surge/issues/3582 | Oortseam | 0:20:27 | 903 days, 17:52:53 | None |
| Ardour export has clicks | https://github.com/surge-synthesizer/surge/issues/3581 | magnetophon | 0:20:15 | 80 days, 9:42:00 | None |
| implement vst3 info.stepCount | https://github.com/surge-synthesizer/surge/issues/3577 | baconpaul | None | 120 days, 22:25:34 | None |
| Oscillator mute automation is not played back in Ardour | https://github.com/surge-synthesizer/surge/issues/3576 | magnetophon | 0:02:47 | 2:40:31 | None |
| Deferred Asset Manager | https://github.com/surge-synthesizer/surge/issues/3573 | baconpaul | 361 days, 22:16:07 | 362 days, 0:31:53 | None |
| Support for subfolders in FX presets | https://github.com/surge-synthesizer/surge/issues/3570 | mkruselj | 0:15:22 | 197 days, 23:05:11 | None |
| AirWindows into Config.xml | https://github.com/surge-synthesizer/surge/issues/3569 | baconpaul | 1:39:23 | 5:33:28 | None |
| "Add modulation from..." option should not include voice-level modulators for Scene parameters | https://github.com/surge-synthesizer/surge/issues/3564 | VincyZed | 0:01:19 | 6:18:40 | None |
| FX value overlays missing: some parameters of some FX in some slots | https://github.com/surge-synthesizer/surge/issues/3563 | j5v | 4:49:27 | 245 days, 15:22:23 | None |
| Can we implement this wavefolder? | https://github.com/surge-synthesizer/surge/issues/3562 | baconpaul | 4:36:48 | 279 days, 10:38:03 | None |
| macro values are not restored in ardour | https://github.com/surge-synthesizer/surge/issues/3561 | magnetophon | 1 day, 17:41:44 | 1 day, 22:43:24 | None |
| Surge segfaults ardour on store | https://github.com/surge-synthesizer/surge/issues/3560 | magnetophon | 0:26:57 | 245 days, 13:55:19 | None |

_This report was generated with the [Issue Metrics Action](https://github.com/github/issue-metrics)_
Search query used to find these items: `repo:surge-synthesizer/surge is:issue created:2021-01-01..2021-04-30 -reason:"not planned"`
